util/Migrator/Migrator.csproj

```diff
@@ -9,6 +9,10 @@
     <EmbeddedResource Include="DbScripts\**\*.sql" />
   </ItemGroup>
 
+  <ItemGroup>
+    <None Remove="DbScripts\2020-03-26_00_CipherSoftDelete.sql" />
+  </ItemGroup>
+
```

Comment:
This gets added whenever you copy/paste. Can be reverted since we have the above wildcard: `<EmbeddedResource Include="DbScripts\**\*.sql" />`